src/modules/ROOT/pages/changelog/202303/03.json

```
@@ -0,0 +1,93 @@
+      ],
+      "why": [
+        "to prevent loosing changes that have't been saved yet."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/1051"
+      ]
+    },
+    {
+      "what": [
+        "the option to add custom styles to all of the web applications. The styles can be added via the user settings in {APP_NAME_ADMIN}."
+      ],
+      "why": [
+        "to provide the user with more freedom to style the applications as they see fit."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/1262"
+      ]
+    }
+  ],
+  "changed": [
+    {
+      "what": [
+        "the provided parameter in the url of a page for selecting one or more records in a record list in {APP_NAME_COMPOSE}."
+      ],
+      "why": [
+        "previously a value wasn't provided, whereas now it is."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/1244"
+      ]
+    },
+    {
+      "what": [
+        "general UI/UX improvements:",
+        "* Hid the button for editing a namespace from the public view in the sidebar when the option to show that namespace isn't visible.",
+        "* Highlighted the row of a translation when viewing subtitle/description namespace translations as it's done everywhere.",
+        "* The styles in file page block so when a gallery of images is viewed, the positioning of each image won't be on top of the previous image's name.",
+        "* The color of the sidebar text items and the functionality to show the text item on hover.",
+        "* Overlapping of the 'more' menu in a record list in {APP_NAME_COMPOSE}."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/1254",
+        "https://github.com/cortezaproject/corteza/pull/1255",
+        "https://github.com/cortezaproject/corteza/pull/1244",
+        "https://github.com/cortezaproject/corteza/pull/1274",
+        "https://github.com/cortezaproject/corteza/commit/5fa587fad20d024fee5e85a0bf533e89001429cd"
+      ]
+    }
+  ],
+  "fixed": [
+    {
+      "what": [
+        "not being able to view a map if the value for latitude or longitude isn't provided."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/1032"
+      ]
+    },
+    {
+      "what": [
+        "error in console whe trying to add a module constraint without providing a value."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/1281"
+      ]
+    },
+    {
+      "what": [
+        "unsaved changes detection error being triggered after a namespace has been saved and you want to visit it."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/commit/aa7cdf28e24263c858ad891b1cf0912596e278da"
+      ]
+    },
+    {
+      "what": [
+        "the labels updated At and deleted At not showing if a {APP_NAME_ADMIN} application is updated or deleted."
```

```suggestion
        "the labels updated at and deleted at not showing if a {APP_NAME_ADMIN} application is updated or deleted."
```

Not sure I quite understand this; can you try and explain it more?